pkg/template/apis/template/validation/validation.go

```diff
@@ -79,6 +80,17 @@ func ValidateTemplateInstance(templateInstance *templateapi.TemplateInstance) (a
 		for _, msg := range oapi.GetNameValidationFunc(uservalidation.ValidateUserName)(templateInstance.Spec.Requester.Username, false) {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("spec.requester.username"), templateInstance.Spec.Requester.Username, msg))
 		}
+		if templateInstance.Spec.Requester.Groups != nil {
+			// unlike users, the system groups may have colons in them, starting with "system:", and will be automatically appended to users group lists
+			for _, group := range templateInstance.Spec.Requester.Groups {
+				if strings.HasPrefix(group, "system:") {
```

Comment:
the validation function used below was complaining about a ":" existing in the group name ... but per you comment above, I'll be deleting this section anyway